admin/src/js/services/version.js

```diff
@@ -31,12 +31,36 @@ angular.module('services').factory('Version',
         }
 
         return version;
-      } else {
-        return {};
       }
     };
 
+    var compare = function(version1, version2) {
+      for (var part of ['major', 'minor', 'patch']) {
```

Comment:
Pretty sure `for .. of` is an es6 feature, which I assume the admin app can't have?

Reply:
Added in Chrome 38 yeah. I think it's fine, this code isn't going to be running on the super old tecnos, but because we haven't made that official yet I'll change it to an older style